src/DotNetEnv/IValue.cs

```diff
@@ -53,18 +53,5 @@ public ValueCalculator (IEnumerable<IValue> values)
             // because it is the state of the world at the moment that this value is calculated
             Value = string.Join(string.Empty, values.Select(val => val.GetValue()));
         }
-
```

Comment:
This is the main goal for me: to get rid of this ickiness.

Reply:
I did not remove the Trim at first to have less changes.
But removing it at a later point was intended by me too, so go for it ;)